Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: test 
+test: check-test-env fmt vet
+	go test ./... -v -parallel 8
+
+.PHONY: fmt 
+fmt: 
+	go fmt ./...
+
+.PHONY: vet
+vet:
+	go vet ./...
```

Comment:
We also have `gofumpt` (instead of `gofmt`) and `govet` in the `golangci-lint` configuration, so not sure if these are needed here. One alternative would be to run `golangci-lint` from the `Makefile`.

Reply:
I'll remove those targets. I guess idea was to just do some quick tests before running tests but as we have already golangci-lint config and pre-commit setup that's not necessary